charts/sn-platform-slim/templates/bookkeeper/bookkeeper-cluster.yaml

```diff
@@ -38,6 +38,9 @@ metadata:
 {{ toYaml . | nindent 4 }}
 {{- end }}
 spec:
+{{- if not .Values.initialize }}
```

Comment:
If initialize is set explicitly to false, what will happen, will initialize be overridden?

Reply:
If the initialize is set as false, then the init job for `initialize-cluster-metadata` command won't be executed.